examples/data-sources/env0_git_token/data-source.tf

```diff
@@ -0,0 +1,12 @@
+resource "env0_git_token" "git_token_sample" {
+  name  = "name"
+  value = "value"
+}
+
+data "env0_git_token" "git_token_sample" {
```

Comment:
Can you please also add an example for fetching this data resource by name?

Reply:
@yaronya 
078cc22530fccc8e51a7043f84eb7b6883d15a96